---a/net/minecraft/world/gen/layer/GenLayerRemoveTooMuchOcean.java
+++b/net/minecraft/world/gen/layer/GenLayerRemoveTooMuchOcean.java
@@ -1,17 +1,16 @@
 package net.minecraft.world.gen.layer;
 
-public class GenLayerRemoveTooMuchOcean extends GenLayer
-{
+public class GenLayerRemoveTooMuchOcean extends GenLayer {
+
     private static final String __OBFID = "CL_00000564";
 
-    public GenLayerRemoveTooMuchOcean(long p_i45480_1_, GenLayer p_i45480_3_)
-    {
+    public GenLayerRemoveTooMuchOcean(long p_i45480_1_, GenLayer p_i45480_3_) {
         super(p_i45480_1_);
         this.parent = p_i45480_3_;
     }
 
-    public int[] getInts(int areaX, int areaY, int areaWidth, int areaHeight)
-    {
+    public int[] getInts(int areaX, int areaY, int areaWidth, int areaHeight) {
         int i1 = areaX - 1;
         int j1 = areaY - 1;
         int k1 = areaWidth + 2;
@@ -19,22 +19,18 @@
         int[] aint = this.parent.getInts(i1, j1, k1, l1);
         int[] aint1 = IntCache.getIntCache(areaWidth * areaHeight);
 
-        for (int i2 = 0; i2 < areaHeight; ++i2)
-        {
-            for (int j2 = 0; j2 < areaWidth; ++j2)
-            {
+        for (int i2 = 0; i2 < areaHeight; ++i2) {
+            for (int j2 = 0; j2 < areaWidth; ++j2) {
                 int k2 = aint[j2 + 1 + (i2 + 1 - 1) * (areaWidth + 2)];
                 int l2 = aint[j2 + 1 + 1 + (i2 + 1) * (areaWidth + 2)];
                 int i3 = aint[j2 + 1 - 1 + (i2 + 1) * (areaWidth + 2)];
                 int j3 = aint[j2 + 1 + (i2 + 1 + 1) * (areaWidth + 2)];
                 int k3 = aint[j2 + 1 + (i2 + 1) * k1];
                 aint1[j2 + i2 * areaWidth] = k3;
-                this.initChunkSeed((long)(j2 + areaX), (long)(i2 + areaY));
+                this.initChunkSeed((long) (j2 + areaX), (long) (i2 + areaY));
 
-                if (k3 == 0 && k2 == 0 && l2 == 0 && i3 == 0 && j3 == 0 && this.nextInt(2) == 0)
-                {
+                if (k3 == 0 && k2 == 0 && l2 == 0 && i3 == 0 && j3 == 0 && this.nextInt(2) == 0) {
                     aint1[j2 + i2 * areaWidth] = 1;
                 }
             }
@@ -40,4 +40,4 @@
 
         return aint1;
     }
-}
+}
